### DISCUSSION TITLE
Userflow scores

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#15352] [Ideas] [answered]
Can lighthouse add a feature to calculate the overall lighthouse score for userflows, like user logging in and applying for a product which includes several navigation, timespan and snapshots. So that we can report a single score for each userflows(login and apply for product, create customer profile etc)?

**Comment by adamraine** [comment]
We don't have any plans to create an overall user flow score. We may make changes to the individual timespan/snapshot scores in the future though #14559.

**Reply by SivasubramanianV** [reply]
Thanks @adamraine ,
> #14573
 i get 21/24 for timespan which includes a soft nav. So does the score include audits related to CLS, INP and TBT? Currently i use following  score for Timespan/snapshot performance score. 

(lhr.categories.performance.score)*100 from the json report that i get from flow.createFlowResult()

Does it take into account CLS, INP and TBT and calculate the score? It gives me 0.83. Also if i use the plugin for softnav, will the lhr.categories.performance.score reflect the LCP, FCP of the softnav. I understand TTI and SI are still not part of Softnav timespan. Still just wanted to give a score to every timespan and snapshot so that we can report something. As of now can i use this to report it for understanding the performance?

**Reply by adamraine** [reply, marked as answer]
> Does it take into account CLS, INP and TBT and calculate the score? It gives me 0.83

That score value takes into account CLS and TBT (but not INP). We intentionally don't display it and should probably update the score value as part of https://github.com/GoogleChrome/lighthouse/issues/14559.

> Also if i use the plugin for softnav, will the lhr.categories.performance.score reflect the LCP, FCP of the softnav

No. The plugin will not affect the existing performance category.

> As of now can i use this to report it for understanding the performance?

I would caution against using the score value since it isn't referenced in the flow report and it doesn't account for INP. Stay tuned on https://github.com/GoogleChrome/lighthouse/issues/14559 as we may update the scoring for timespan in the near future.

If you really need a single score, then `lhr.categories.performance.score` can be an adequate summary of just CLS and TBT.

**Reply by SivasubramanianV** [reply]
Thanks Adam. Any plans to include SI also to soft nav timespan? Or SI is not applicable for softnav? Also in latest lh reports i dont see TTI @adamraine

**Reply by adamraine** [reply]
SI will likely be included for soft navigations.

TTI was removed from the report in v10.
